src/App.css

```diff
@@ -0,0 +1,142 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #61dafb;
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+body {
+  background-color: rgb(228, 227, 222);
+}
+
+nav{
+  display: flex;
+  justify-content: space-around;
+  text-transform: uppercase;
+  background: dimgrey;
+  color: aliceblue;
+
+  border-bottom: 5px solid rgb(250, 107, 24);
+}
+ul a {
+  
+  color: rgb(235, 134, 18);
+}
+
+nav  .logo{
+  color: rgb(223, 115, 14);
+  float: left;
+  font-size: 50px;
+  text-decoration: none;
+}
+
+nav .links{
+  display: flex;
+  justify-content: space-between;
+  align-items:center;
+  text-decoration: none;
+}
+
+
+nav li {
+
+  display: flex;
+  margin:10px;
+  font-size: 20px;
+  
+  
+  
```

Comment:
the same comment as above. Code should be consistent. Please check other places on the same comment. Consistent I mean written in the same styling. So if in one place we use couple empty lines then why in another similar place of code we do not use them? We should stick to the same approach and either use empty lines if they needed either do not use them in the similar places of code